.github/workflows/integration_alliance_workflow_releaser.yml

```
@@ -0,0 +1,103 @@
+name: Integration Alliance Workflow Releaser
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  AWS_ENV: ${{ secrets.AWS_ENV }}
+  AWS_STS_PLUGIN_S3_ROLE: ${{ secrets.AWS_STS_PLUGIN_S3_ROLE }}
+  AWS_STS_PLUGIN_S3_EXTERNAL_ID: ${{ secrets.AWS_STS_PLUGIN_S3_EXTERNAL_ID }}
+
+jobs:
+  integration_alliance_hub_workflow_releaser:
+    runs-on: ubuntu-latest
+
+    inputs:
+      SILENT_UPLOAD:
+        description: 'Whether or not to update dates when releasing'
+        required: true
+        default: false
+        type: boolean
+      ICON_TYPE:
+        description: 'Type(workflow or template) to release'
+        required: true
+        default: 'workflow'
+        allowedValues: ['workflow', 'template']
+      ICON_WORKFLOW:
+        description: 'Workflow to release by its folder name from the respective GitHub repo, e.g. Malicious_Hash_Remediation_with_Cb_Response'
+        required: false
+      ICON_RELEASE_ALL:
+        description: 'Release ALL files of one type to the Hub.'
+        required: true
+        default: false
+        type: boolean
+      ICON_RELEASE_DIFF:
+        description: 'Diff the source and target branch to determine which workflows to release.'
+        required: true
+        default: true
+        type: boolean
+      ENVIRONMENT:
+        description: 'Environment to release to'
+        required: true
+        default: 'prod'
+        allowedValues: ['prod', 'staging']
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
```

Here is newer version as well 😄 
```suggestion
        uses: actions/setup-python@v4
```